auction-server/src/auction.rs

```diff
@@ -986,7 +1031,15 @@ pub async fn handle_bid_svm(
         submit_bid_instruction,
     )?;
 
-    verify_signatures_svm(&bid, &chain_store.express_relay_svm.relayer.pubkey())?;
+    let bytes_permission_key = Bytes::from(&permission_key.0);
+    verify_signatures_svm(
+        store_new,
+        chain_store,
+        &bid,
+        &chain_store.express_relay_svm.relayer.pubkey(),
+        &bytes_permission_key,
+    )
+    .await?;
```

Comment:
we should verify that the wallet bids also include another instruction to the swap program with the appropriate accounts and fields. Please add a TODO for it here since we don't have the program yet.